defaults.lua

```diff
@@ -33,7 +33,7 @@ DGLOBAL_CACHE_SIZE_MINIMUM = 1024*1024*16
 DGLOBAL_CACHE_FREE_PROPORTION = 0.4
 
 -- maximum cache size
-DGLOBAL_CACHE_SIZE_MAXIMUM = 1024*1024*64
+DGLOBAL_CACHE_SIZE_MAXIMUM = 1024*1024*512
```

Comment:
Actually I said 1024, since it's either 0.4 of total or the other max. But 512 should be fine as a general default.

Reply:
Yeah, that was half being very conservative, half having a hard time seeing where we could actually use > 512MB of cache ;p.

Reply:
Sidebar: It seems like CircleCI is having yet another sad right now...

Reply:
> half having a hard time seeing where we could actually use > 512MB of cache ;p.

My old old phone has 4 GB RAM, my new phone has 8 GB RAM, my PC has 32 GB RAM, …

Reply:
My bad, I meant "what for", not where as in platforms ;).

Reply:
But sure, on my H2O the 0.4 will limit it to 200 or so (unless I happen to have adjusted that, but iirc I only changed the DGLOBAL_CACHE_SIZE_MAXIMUM to something like 200 or 256).

Reply:
FWIW, on the H2O:

```
08/19/21-19:24:31 DEBUG Allocating a 133MB budget for the global document cache
```

(Remember, it's 40% of *available* memory ;)).

Reply:
> My bad, I meant "what for", not where as in platforms ;).

A few pages add up pretty quickly, especially combined with zoom.

I even complained about Evince a few years ago.
https://fransdejonge.com/2017/10/fix-minuscule-zoom-limit-in-evince/

Going for less than this just didn't do it for me:
```
$ gsettings get org.gnome.Evince page-cache-size
uint32 1024
```

Reply:
> (Remember, it's (roughly) 40% of available memory ;)).

Which is a good thing, but yes, I did forget about that for the moment. :-P